src/Directory.Build.props

```diff
@@ -24,8 +24,7 @@
 <!-- API Compat -->
   <PropertyGroup>
     <RunApiCompat Condition="'$(RunApiCompat)' == ''">$(IsStableProject)</RunApiCompat>
-    <ToolHostCmd Condition="'$(OS)' != 'Windows_NT'">$(ToolsDir)dotnetcli/dotnet</ToolHostCmd>
```

Comment:
Oh, that's interesting. So we were assuming that if it was windows, *then* we wouldn't need to use the tools directory. Fun stuff. 😄

Reply:
Yes, on the CI windows machines most likely it was installed by default, but not on the official build machines.